### PR TITLE
Styleguide and & UI naming updates

### PR DESCRIPTION
**Pull request by karensawrey**
- Renamed Security section to Audit section
- Fixed an outdated link to the file containing sidebar code
- Expanded the section on capitalization in the STYLEGUIDE to make it as clear as possible.
- Added a new section on anchor links and multi-level anchor links

### CODE REVIEW

File: styleguide/STYLE.md [hunk 1]
```diff
@@ -45,11 +45,11 @@ Keep your writing consistent with itself and other docs. This means abbreviation
 
 ### Title capitalisation
 
-Use Title case in page headings:
+Use Title case in page headings (\#-level - H1):
```
Comment (plaindocs) [inline styleguide/STYLE.md:48]:
```suggestion
Use Title case in H1 page headings `#Heading`:
```

I think I prefer something like this, what about you?

File: styleguide/STYLE.md [hunk 2]
```diff
@@ -45,11 +45,11 @@ Keep your writing consistent with itself and other docs. This means abbreviation
...
 *"The Quick Brown Fox Jumps Over the Lazy Dog."*
 Also known as headline style and capital case. All words capitalized, except for certain subsets defined by rules that are not universally standardized, often minor words such as "the" (as above), "of", or "and". The standardization is only at the level of house styles and individual style manuals. (See Headings and publication titles.) A simplified variant is start case, where all words, including articles, prepositions, and conjunctions, start with a capital letter.
 
-Use Sentence case in section headings:
+Use Sentence case in section headings (\#\#-H2,\#\#\#-H3, \#\#\#\#-H4, etc.):
```
Comment (plaindocs) [inline styleguide/STYLE.md:52]: As above.

File: styleguide/STYLE.md [hunk 3]
```diff
@@ -171,7 +171,7 @@ Our docs website is a custom build. This section gives some guidance on working
 ### Adding and naming new documentation pages
 To add a new documentation page, create it as a *.md.erb file. Give it a lowercase name, separate words using underscores.
 To add the new page to the documentation sidebar on https://buildkite.com/docs, add the corresponding entry to
-`app/views/layouts/application.html.erb` with a description (e.g. `"G Cloud Identity", 'integrations/sso/g-cloud-identity'` ).
+`app\views\layouts\_sidebar.html.erb` with a description (e.g. `"G Cloud Identity", 'integrations/sso/g-cloud-identity'` ).
```
Comment (plaindocs) [inline styleguide/STYLE.md:174]: I think we use `/` for paths in the rest of the docs, let's stick to that.

File: styleguide/STYLE.md [hunk 4]
```diff
@@ -197,6 +197,15 @@ Internal links need to start with `/docs`, for example:
 Read more about [environment variables](/docs/pipelines/environment-variables)
 ```
 
+### Anchor links  
+
+To use an anchor link where you need to link to an H2-level heading, append the section's name to the main page link, for example:
```
Comment (plaindocs) [inline styleguide/STYLE.md:202]: Worth adding that these are generated automatically from the title, and viewable in the # that appears when you mouse over the heading?

**Review by plaindocs — APPROVED** [review]
Nice! I've made a few suggestions.

File: styleguide/STYLE.md (hunk 3 above)
Comment (karensawrey) [reply styleguide/STYLE.md:174]: Oh, this is a weird serial typo. Fixing.

File: styleguide/STYLE.md (hunk 4 above)
Comment (karensawrey) [reply styleguide/STYLE.md:202]: Yes, adding.

**Comment by karensawrey** [comment]
@plaindocs please take another look at the changes & I'll ship!

**Pull request merged**